health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/DataUploadWrapper.js

```
@@ -0,0 +1,79 @@
+    "HCM_UPLOAD_FACILITY",
+    "HCM_UPLOAD_FACILITY_MAPPING",
+    "HCM_UPLOAD_USER",
+    "HCM_UPLOAD_USER_MAPPING",
+    "HCM_UPLOAD_TARGET",
+    "HCM_SUMMARY",
+  ];
+  const mappingCategories = ["HCM_UPLOAD_FACILITY_MAPPING", "HCM_UPLOAD_USER_MAPPING"];
+  const [currentStep, setCurrentStep] = useState(1);
+  const currentCategories = categories?.[currentStep - 1];
+  const [key, setKey] = useState(() => {
+    return currentKey ? parseInt(currentKey) : 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)